Analytics/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -630,32 +619,17 @@ - (NSUInteger)maxBatchSize
     return 100;
 }
 
-- (NSURL *)userIDURL
-{
-    return SEGAnalyticsURLForFilename(@"segmentio.userId");
-}
-
-- (NSURL *)queueURL
-{
-    return SEGAnalyticsURLForFilename(@"segmentio.queue.plist");
-}
-
-- (NSURL *)traitsURL
-{
-    return SEGAnalyticsURLForFilename(@"segmentio.traits.plist");
-}
-
 - (NSString *)getUserId
 {
-    return [[NSUserDefaults standardUserDefaults] valueForKey:SEGUserIdKey] ?: [[NSString alloc] initWithContentsOfURL:self.userIDURL encoding:NSUTF8StringEncoding error:NULL];
+    return [[NSUserDefaults standardUserDefaults] valueForKey:SEGUserIdKey] ?: [self.analytics.storage stringForKey:kSEGUserIdFilename];
 }
 
 - (void)persistQueue
 {
 #if TARGET_OS_TV
-    [[NSUserDefaults standardUserDefaults] setValue:[self.queue copy] forKey:SEGQueueKey];
+    [self.analytics.storage setArray:[self.queue copy] forKey:SEGQueueKey];
```

Comment:
@f2prateek this is pretty in-elegant because the name of keys are different depending on whether we are using user defaults or files. I'd be interested in getting a proper migration system setup where we can make this consistent so no need to do these crazy `#if` statements.